analyses/focal-cn-file-preparation/01-prepare-cn-file.R

```diff
@@ -76,22 +76,31 @@ process_annotate_overlaps <- function(cnv_df,
   cnv_gr <- cnv_df %>%
     GenomicRanges::makeGRangesFromDataFrame(keep.extra.columns = TRUE,
                                             starts.in.df.are.0based = FALSE)
-
-  # Create a data.frame with the overlaps between the CNV file and hg38 genome
-  # annotations
-  overlaps <- IRanges::mergeByOverlaps(cnv_gr, tx_exons)
-
-  # Create a data.frame with selected columns from the `autosome_overlaps`
+  
+  # Find the overlaps between the CNV file and GENCODE v27 filtered hg38
+  # annotations file (converted to bed file in `run-prepare-cn.sh`).
+  overlaps <- GenomicRanges::findOverlaps(cnv_gr, cds_gr)
+  
+  # Carry over samples to `cnv_gr` object
+  cnv_gr$org_id <- rep(NA, length(cnv_gr))
```

Comment:
Unless I'm misunderstanding this, it may be better to name this something more related to what it is, like `gene_id`.